HAB5/hab5.tex

```diff
@@ -0,0 +1,404 @@
+An example could be the linking of IMU attitude data to images at the corresponding time, which in turn may allow for correction of apparent areas due to the angle of the camera with respect to the horizon. 
+This in turn prepares SPEX for projects requiring this correction be done on-flight, and for other projects such as horizon detection which could ``train'' on-flight by comparing analyzed data to known IMU readings. 
+To this end the synchronization of clocks may be pursued. 
+Another consistent challenge has been the processing of information on a stream, without information being skipped or resampled. 
+Raw and processed information should be saved for later evaluation and iteration of systems.
+
+Significant departures from prior procedures in this proposition include the rigorous testing of components and assembled hardware before flight, training/tuning of algorithms where needed and the inclusion of metrics to detect components going offline, development and propagation of errors. 
+As SPEX moves to consider CV modules, it is expected that the development of training knowledge in pre-flight testing will become crucial. 
+Among the tests are those for: camera alignment; camera stability; spectral response/quantum efficiency; software processing of simulations; payload NDVI/visual functioning while on ground; an electronics functional test with specific focus on power systems and APRS reliability; electronics environmental testing; integrated system testing.
+Time-stamping of information received by different processes may be considered. 
+It is also desired that metrics be included for post-flight analysis.
+
+Among the physical requirements for the payload, it is required that the HAB vehicle and payload system be better capable of protecting from environmental factors such as humidity, winds and temperature. Specific to this payload, the alignment of cameras is a question needing further exploration. An angle and separation optimal for the combination of data acquisition and securedness is required. Vibration or displacement during flights can significantly impair or corrupt all information gathered. Fixing this hurdle is critical for future projects which may look to infer distance data with a combination of two cameras or a radar altimeter setup. While the project doesn't expect that these will all be met as results, it is firmly held that the steps taken to implement these goals as best as possible will transformatively improve SPEX's advanced payload capabilities.
+
+\subsection{Milestones}
+Development and fabrication milestones are listed in \autoref{milestones-table}.
+\begin{table}[!ht]
+  \centering
+  \caption{Milestones to design and build HAB5}
+
+\begin{minipage}{.45\textwidth}
+  \noindent
+  \begin{tabular}{@{}ll@{}}
+
+  \label{milestones-table}
+    \textbf{Task} & \textbf{Complete Date} \\
```

Comment:
It would be nice to see more sensor testing milestones in the Fall semester, since that is the bulk of the work. Is there a mobile test that can be done?

Reply:
@keshavadhyay is there a more focused way to say "sensor acquisition, readout config" and "camera setup" that would be milestones for achieving the objectives specific to this mission?